sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/TableTypeMapping.java

```diff
@@ -27,18 +27,18 @@
    * @param clientTypeName
    * @return
    */
-  String mapToHiveType(String clientTypeName);
+  public String mapToHiveType (String clientTypeName);
 
   /**
    * Map hive's table type name to client's table type
-   * @param hiveTypeName
+   * @param clientTypeName
    * @return
```

Comment:
I guess this is pulled in but hiveTypeName is the right param in the doc not clientTypeName